docs/examples/analysis_apps.rst

```diff
@@ -0,0 +1,230 @@
+ROOT based analysis applications
+================================
+
+The ACTS examples come with a certain variety of ROOT based validation and performance writers, 
+whose output can be use to understand various aspects of the reconstruction in more detail.
+
+The building of these applications can be switched on by setting ``ACTS_BUILD_ANALYSIS_APPS=On``,
+which requires (on top of the Core dependencies, also ROOT for the analysis code).
```

Comment:
The parenthesis in that sentence are a bit weird.

Reply:
Fixed now.